suite-native/device-manager/src/components/WalletItem.tsx

```diff
@@ -7,36 +7,52 @@ import { Icon } from '@suite-common/icons';
 import { TrezorDevice } from '@suite-common/suite-types';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { selectDevice, selectDeviceByState } from '@suite-common/wallet-core';
+import { FiatAmountFormatter } from '@suite-native/formatters';
+import { selectDeviceTotalFiatBalanceNative } from '@suite-native/device';
 
 type WalletItemProps = {
     deviceState: NonNullable<TrezorDevice['state']>;
     onPress: () => void;
     isSelectable?: boolean;
 };
 
-const walletItemStyle = prepareNativeStyle<{ isSelected: boolean }>((utils, { isSelected }) => ({
-    paddingHorizontal: utils.spacings.medium,
-    justifyContent: 'space-between',
-    alignItems: 'center',
-    height: 60,
-    gap: 12,
-    backgroundColor: utils.colors.backgroundSurfaceElevation1,
-    borderWidth: utils.borders.widths.small,
-    borderRadius: 12,
-    borderColor: utils.colors.borderElevation1,
-    extend: {
-        condition: isSelected,
-        style: {
-            borderWidth: utils.borders.widths.large,
-            borderColor: utils.colors.borderSecondary,
-        },
-    },
-}));
+type WalletItemStyleProps = { isSelected: boolean; isSelectable: boolean };
+
+const walletItemStyle = prepareNativeStyle<WalletItemStyleProps>(
+    (utils, { isSelected, isSelectable }) => ({
+        justifyContent: 'space-between',
+        alignItems: 'center',
+        height: 60,
+        gap: 12,
+        borderRadius: 12,
+        borderColor: utils.colors.borderElevation1,
+        extend: [
+            {
+                condition: isSelected,
+                style: {
+                    borderWidth: utils.borders.widths.large,
+                    borderColor: utils.colors.borderSecondary,
+                },
+            },
+            {
+                condition: isSelectable,
+                style: {
+                    paddingHorizontal: utils.spacings.medium,
+                    backgroundColor: utils.colors.backgroundSurfaceElevation1,
+                    borderWidth: utils.borders.widths.small,
+                },
+            },
+        ],
+    }),
+);
 
 export const WalletItem = ({ deviceState, onPress, isSelectable = true }: WalletItemProps) => {
     const { applyStyle } = useNativeStyles();
     const device = useSelector((state: any) => selectDeviceByState(state, deviceState));
     const selectedDevice = useSelector(selectDevice);
+    const fiatBalance = useSelector((state: any) =>
```

Comment:
For the fiat value on dashboard we use `getTotalFiatBalance` that includes tokens, right? So it can lead to mismatch. Is that intentional?